src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureColor {
```

Comment:
The class name `Rectangle` is appropriate as it is a noun. However, ensure that all other class and interface names in your project also follow the convention of using nouns and not verbs.